torchaudio/models/rnnt.py

```diff
@@ -377,12 +377,17 @@ class _Joiner(torch.nn.Module):
     Args:
         input_dim (int): source and target input dimension.
         output_dim (int): output dimension.
+        use_tanh (bool, optional): if ``True``, uses Tanh rather ReLU activation. (Default: ``False``)
+
     """
 
-    def __init__(self, input_dim: int, output_dim: int) -> None:
+    def __init__(self, input_dim: int, output_dim: int, use_tanh: bool = False) -> None:
         super().__init__()
         self.linear = torch.nn.Linear(input_dim, output_dim, bias=True)
-        self.relu = torch.nn.ReLU()
+        if use_tanh:
+            self.activation = torch.nn.Tanh()
+        else:
+            self.activation = torch.nn.ReLU()
```

Comment:
then, if we use `activation`, we can update this to raise an exception in the default case to prevent users from choosing an activation function they didn't intend to choose.

example:
https://github.com/pytorch/audio/blob/66185e0074d06330dff4ea84cbface3b83312ec4/torchaudio/models/emformer.py#L39-L47